Src/CSharpier.Tests/TestFiles/TypeParameterConstraintClause/TypeParameterConstraintClauses.cst

```diff
@@ -33,4 +51,7 @@ class ClassName<N, C, T, TT, L>
     where C : class
     where T : IComparable
     where TT : IList<N>
-    where L : class, new() { }
```

Comment:
Same here. You could just duplicate this, so we still have a "class with type constraints but no body" test, which could be different than the `interface` or `struct` case.